src/Components/App.jsx

```diff
@@ -1,9 +1,17 @@
-import './App.css';
+import './App.scss';
+import { Header } from './Header/Header';
+import { Footer } from './Footer/Footer';
+import { Main } from './Main/Main';
+import { BrowserRouter } from 'react-router-dom';
 
 export const App = () => {
   return (
-    <div className="App">
-      Social Network
-    </div>
+    <BrowserRouter>
+      <div className="wrapper">
```

Comment:
I strongly recommend to use sass modules, it will incapsulate your styles and they will affect only desire component =)

Reply:
Done!